sitegen.sln

```diff
@@ -2,6 +2,8 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Sitegen", "src\Sitegen\Sitegen.csproj", "{A407BBF4-56E2-4720-80B0-D7B4AFD51F35}"
 EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Sitegen.Tests", "Sitegen.Tests\Sitegen.Tests.fsproj", "{B260256E-34CF-4ABA-8E55-A1C5FEA9E940}"
+EndProject
```

Comment:
Hmm, we should probably add `dotnet test` or something to the CI config also, to ensure that we don't just build the solution but actually run all tests in CI.

Reply:
Amended to existing commit, resolving.